Codigo/CustomScenarios.bas

```diff
@@ -123,9 +128,25 @@ On Error GoTo PrepareNewEvent_Err:
     Select Case EventType
         Case e_EventType.NpcHunt
             Set GenericGlobalLobby.scenario = New ScenarioHunt
+        Case e_EventType.DeathMatch
+            Set GenericGlobalLobby.scenario = New ScenarioDeathMatch
```

Comment:
missing `case else`

Reply:
there is nothing to do in an else case, this is intended, its  fine to do nothing if there is no event attached to a lobby